core/src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -281,6 +281,12 @@ class UserService(
     if (roles.nonEmpty) Some(mapCloudPlatformAndPolicies(p, billingProfile, roles, workspaceManagerDAO)) else None
   )
 
+  def getBillingProjectById(id: UUID): Future[Option[RawlsBillingProjectResponse]] =
+    billingRepository.getBillingProjectById(id).flatMap {
+      case Some(billingProject) => getBillingProject(billingProject.projectName)
```

Comment:
I would recommend extracting the logic in `getBillingProject` that's required to construct the full `RawlsBillingProjectResponse` into a shared helper method rather than calling the full method here and adding a second round trip to the database.

Reply:
Fixed: https://github.com/broadinstitute/rawls/pull/3205/commits/a67f8bf57cbcdcc828805559a4f3c2da04e87ce0